R/crosstab.proportion.R

```diff
@@ -0,0 +1,27 @@
+proportions.frame <- function(values, ...){
```

Comment:
Add bare minimum docstring.  Should have:
* Basic description
* Param-values
* Return value